compose/fastapi/start

```diff
@@ -8,6 +8,9 @@ set -o nounset
 export UVICORN_HOST="0.0.0.0"
 export UVICORN_PORT=80
 
-opentelemetry-instrument \
-    --logs_exporter otlp \
-    uvicorn src.main:app --reload --host ${UVICORN_HOST} --port ${UVICORN_PORT} --proxy-headers
+#opentelemetry-instrument \
+#    --logs_exporter otlp \
+#    uvicorn src.main:app --reload --host ${UVICORN_HOST} --port ${UVICORN_PORT} --proxy-headers
+
```

Comment:
Are we using opentelemetry? Maybe we can remove all opentelemetry from dependencies

Reply:
DataDog is replacing it.  I didn't want to remove it from the dependencies just yet.  We are still in a trial period.